pallets/storage-provider/src/lib.rs

```diff
@@ -1231,9 +1238,9 @@ pub mod pallet {
                 }
 
                 log::info!(target: LOG_TARGET, "block: {:?}, checking storage provider {:?} deadline: {:?}",
-                   current_block,
-                   storage_provider,
-                   current_deadline.idx,
+                    current_block,
```

Comment:
formatting or what heppened here?

Reply:
Yeah, this is formatting. Not sure why this wasn't formatted before 🤷